### ISSUE TITLE
Parallel categories not recognizing dimension columns

### ISSUE THREAD

**Issue by robroc** [plotly/plotly_express#98]
Data frame with two categorical columns and one numerical column being passed to `px.parallel_categories`. Only one column gets visualized, no matter what is passed to `dimensions`: a single column, two columns, or no columns.

Data types changed to `str` and `category`, but no change.

Data:

![px](https://user-images.githubusercontent.com/1219001/59059308-a48c1480-886c-11e9-91b8-52772e6656fd.PNG)

Code:

`px.parallel_categories(occupations_by_year, dimensions=['Main_jobs', 'Election year'], color = 'count')`

Result:

![px2](https://user-images.githubusercontent.com/1219001/59059317-a7870500-886c-11e9-8b91-7b49ce5a109c.PNG)

Python 3.6
plotly==3.10.0
plotly-express==0.3.0

**Comment by nicolaskruchten** [comment]
How many unique values of `main_jobs` are there?

**Comment by robroc** [comment]
About 30 unique values. I was hoping to make each year a node in the chart
and the jobs the bands between them, a bit like a Sankey. Is this possible?

**Comment by nicolaskruchten** [comment]
Yes, but I'll have to generalize the cutoff heuristic, which right now caps the number of values at 20 IIRC. This a little blunt for cases like this where you only have 2 dimensions. It's reasonably if you had e.g. 10 dimensions because in that case you might have up to 20^10 combinations :)

**Comment by jacobbaron** [comment]
I have encountered this issue as well. I have four columns, one of which has ~30 categories. There are about 1100 unique combinations in the data, so not ridiculous, but the column with the 30 categories is left out of the plot.

**Comment by jasonsross** [comment]
I 2nd the desire to set the cutoff of number of categories. Like these charts but I've hit this roadblock and I think I was just barely over the cutoff limit. I'm ok to group my categories down to reduce complexity, but wish to have a little more flexibility in doing so.

**Comment by nicolaskruchten** [comment]
I'll get this fixed in the next version :) I'll move this issue to the plotly.py repo to get it scheduled.
